src/UI/Seller/src/app/buyers/components/buyers/buyer-edit/buyer-edit.component.ts

```diff
@@ -36,30 +41,63 @@ export class BuyerEditComponent {
 
   constructor(
     private buyerService: BuyerService,
+    private ocImpersonationService: OcImpersonationConfigService,
     private router: Router,
     private buyerTempService: BuyerTempService,
     private appAuthService: AppAuthService
   ) {}
 
   updateResourceFromEvent(event: any, field: string): void {
-    const value =
+    let resourceUpdate: ResourceFormUpdate;
+    if(field==="ImpersonatingEnabled") {
+      resourceUpdate= {
```

Comment:
looks like you need to run the linter on these files, spacing is off.

Reply:
Just updated